ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -960,7 +960,6 @@
 				LD_RUNPATH_SEARCH_PATHS = "/usr/lib/swift $(inherited)";
 				LIBRARY_SEARCH_PATHS = (
 					"\"$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)\"",
-					"\"$(TOOLCHAIN_DIR)/usr/lib/swift-5.0/$(PLATFORM_NAME)\"",
```

Comment:
Because keeping that path after updating the npm package `urbanairship-react-native` causes a compilation error for iOS. This was commented in [this issue](https://github.com/urbanairship/ios-library/issues/301).

I think keeping only the line above should be enough.

Reply:
Also, I had updated to `urbanairship-react-native@13.0.0` and not to the latest version to avoid incompatibilities. However, apparently we could (and should) update to a higher version — in this case `urbanairship-react-native@ 14.0.0`.